content/12.raw-data-analysis.md

```diff
@@ -61,6 +62,30 @@ The existing software for DDA and DIA analysis can be further divided into freew
 
 
 ### Analysis of DDA data
+DDA data analysis can be performed in one of the two ways- either with the vendor propreitary format directly analyzed using a propreitary search engine like Mascot, Sequest (through Proteome Discoverer), Paragon (through Protein Pilot), or it can be processed through one of the many freely available search engines or pipelines, for example - MaxQuant, MSGF+, X!Tandem, Morpheus, MSFragger, OMSSA, or Trans-Proteomics pipeline (TPP).
```

Comment:
```suggestion
DDA data analysis either directly uses the vendor proprietary data format directly with a proprietary search engine like Mascot, Sequest (through Proteome Discoverer), Paragon (through Protein Pilot), or it can be processed through one of the many freely available search engines or pipelines, for example, MaxQuant, MSGF+, X!Tandem, Morpheus, MSFragger, and OMSSA. 
Tables 1 and 4 give weblinks and citations for these software tools.
<!-- todo: need to make sure all these are listed in the table and cited -->
<!-- todo: need a paragraph discussing intergration environments like the transproteomics pipeline (TPP) and IDpicker -->
```